README.md

```diff
@@ -0,0 +1,45 @@
+Flatpak manifest for GitHub Destkop app
+=======================================
+
+This repository contains the files to create a Flatpak version of the [GitHub Desktop](https://desktop.github.com/) Git client.
+
+It is based on [this fork](https://github.com/shiftkey/desktop) which contains extra work for better Linux integration, that is, for now, not official.
+
+Installation
+------------
+
+To build and install this Flatpak, you have to [install Flatpak, Flatpak builder and the Flathub repo](https://flatpak.org/setup/). Don't forget to initialize this repo submodules. Then run:
+
+```sh
+flatpak-builder build io.github.shiftey.Desktop.yaml --repo=repo --install --force-clean --install-deps-from=flathub
+```
+
+Once installed, launch GitHub Desktop by running:
+
+```sh
+flatpak run io.github.shiftey.Desktop
+```
+
+Updating `desktop` repo and dependencies
```

Comment:
This probably could be automated to some degree, like here: https://github.com/flathub/io.openrct2.OpenRCT2/blob/master/.github/workflows/beta.yml and https://github.com/OpenRCT2/OpenRCT2/blob/develop/.github/workflows/ci.yml#L252

Reply:
I am going to look into setting a beta branch similar to what openrct 2 is doing